backend/src/test/java/team/teamby/teambyteam/schedule/acceptance/ScheduleAcceptanceTest.java

```diff
@@ -0,0 +1,136 @@
+package team.teamby.teambyteam.schedule.acceptance;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import team.teamby.teambyteam.common.AcceptanceTest;
+import team.teamby.teambyteam.schedule.application.dto.ScheduleRegisterRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+import static team.teamby.teambyteam.fixtures.ScheduleFixtures.팀플_1번_N시간_일정;
+
+@SuppressWarnings("NonAsciiCharacters")
+public class ScheduleAcceptanceTest extends AcceptanceTest {
+
+    private static final String JWT_PREFIX = "Bearer ";
+    private static final String JWT_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiZW1haWwiOiJhYmNkQGdtYWlsLmNvbSIsImlhdCI6MTUxNjIzOTAyMn0.2Amns5eXoGSqLw5KW-i22lH3S85-wfNd3j6Zs2z2Fg4";
+    private static final String REQUEST_TITLE_KEY = "title";
+    private static final String REQUEST_START_DATE_TIME_KEY = "startDateTime";
+    private static final String REQUEST_END_DATE_KEY = "endDateTime";
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Nested
+    @DisplayName("일정 등록 시")
+    class registerSchedule {
+
```

Comment:
[RC]

```suggestion
    @Nested
    @DisplayName("일정 등록 시")
    class RegisterSchedule {

```

클래스명 대문자요!!